hack/update-vendor-licenses.sh

```diff
@@ -216,6 +217,13 @@ for PACKAGE in $(go list -m -json all | jq -r .Path | sort -f); do
     echo "${PACKAGE}, temporarily skipping" >&2
     continue
   fi
+  if [[ "${PACKAGE}" = "k8s.io/klog" ]]; then
```

Comment:
Temporarily add skip rules for work-around.
Upgrade this scripts later.

Reply:
Vendor licenses can't pass?

Reply:
@fisherxu `update-vendor-licenses.sh` now regard path `k8s.io/klog` as a package, in fact it is not package but a path to "k8s.io/klog/v2" . It has been fixed by k8s master repo.

Reply:
>It has been fixed by k8s master repo.

Could we fix it in kubeedge?

Reply:
> It has been fixed by k8s master repo.

Could we fix it in kubedge?

Reply:
@fisherxu There are some errors when directly upgrade this scriptto k8s master version. We could fix later in next pr.

Reply:
@fisherxu  recorded in #2350